client/webserver/site/src/js/markets.js

```diff
@@ -208,18 +208,50 @@ export default class MarketsPage extends BasePage {
     }
   }
 
+  /* showLoaderMsg shows a message at loaderMsg and hides the orderForm in case
+  * dexc does not support an asset.
+  */
+  showLoaderMsg (symbol) {
+    const page = this.page
+    Doc.hide(page.orderForm)
+    Doc.show(page.loaderMsg)
+    page.loaderMsg.textContent = `The asset ${symbol} is not supported.`
```

Comment:
Does it need to say "The asset vtc". Would anyone reasonably assume that VTC is something else besides a blockchain asset? If not, then 

```
 `${symbol.toUpperCase()} is not supported`
```

is better.